kubeflow/common/iap-ingress/base/cluster-role.yaml

```diff
@@ -17,8 +17,10 @@ rules:
 - apiGroups:
   - extensions
   - networking.k8s.io
+  - apps
   resources:
   - ingresses
+  - deployments
```

Comment:
Why do we need to add `apps` and `deployments` permission?

Reply:
That's for fetching the `requestPath`